test/specs/commands/set.js

```diff
@@ -602,11 +602,185 @@ describe('set command', () => {
+														() => {
+															Then(
+																'it should update the config nested variable "api.nodes" to the inputs',
+																then.itShouldUpdateTheConfigNestedVariableToTheInputs,
+															);
+															Then(
+																'it should write the updated config to the config file',
+																then.itShouldWriteTheUpdatedConfigToTheConfigFile,
+															);
+															Then(
+																'it should resolve to an object with message "Successfully set api.nodes to localhost,http://127.0.0.1."',
+																then.itShouldResolveToAnObjectWithMessage,
+															);
+														},
+													);
+												},
+											);
+										},
+									);
+								},
+							);
+							Given('inputs "http://localhost:4000"', given.inputs, () => {
+								Given(
+									'the config file can be written',
+									given.theConfigFileCanBeWritten,
+									() => {
+										Given(
+											'Vorpal is in non-interactive mode',
+											given.vorpalIsInNonInteractiveMode,
+											() => {
+												When(
+													'the action is called with the variable and the inputs',
+													when.theActionIsCalledWithTheVariableAndTheInputs,
+													() => {
+														Then(
+															'it should update the config nested variable "api.nodes" to the inputs',
+															then.itShouldUpdateTheConfigNestedVariableToTheInputs,
+														);
+														Then(
+															'it should write the updated config to the config file',
+															then.itShouldWriteTheUpdatedConfigToTheConfigFile,
+														);
+														Then(
+															'it should resolve to an object with message "Successfully set api.nodes to http://localhost:4000."',
+															then.itShouldResolveToAnObjectWithMessage,
+														);
+													},
+												);
+											},
+										);
+										Given(
+											'Vorpal is in interactive mode',
+											given.vorpalIsInInteractiveMode,
+											() => {
+												When(
+													'the action is called with the variable and the inputs',
+													when.theActionIsCalledWithTheVariableAndTheInputs,
+													() => {
+														Then(
+															'it should update the config nested variable "api.nodes" to the inputs',
+															then.itShouldUpdateTheConfigNestedVariableToTheInputs,
+														);
+														Then(
+															'it should write the updated config to the config file',
+															then.itShouldWriteTheUpdatedConfigToTheConfigFile,
+														);
+														Then(
+															'it should resolve to an object with message "Successfully set api.nodes to http://localhost:4000."',
+															then.itShouldResolveToAnObjectWithMessage,
+														);
+													},
+												);
+											},
+										);
+									},
+								);
+							});
+							Given('inputs ""', given.inputs, () => {
```

Comment:
What about no inputs? I.e. setting an empty array rather than setting an array with one empty string element.

Reply:
As the command `set` inputs, even if not typing anything, it should come back with the `empty array`.